src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/PasswordController.kt

```diff
@@ -51,9 +51,13 @@ class PasswordController(private val userLogic: UserLogic,
     @ResponseBody
     @Throws(TokenException::class)
     fun postResetPassword(@RequestParam("token") token: String,
-                          @RequestParam("password") @Size(min = 6, message = "Password must be at least 6 characters long")
+                          @RequestParam("password")
                           password: String): String {
-        val user = oneTimeTokenManager.validateToken(token) ?: throw TokenException("Token is not valid")
+        if (password.length < 6) {
+            throw HintException("invalidPasswordLength", HttpStatus.BAD_REQUEST)
```

Comment:
I changed this over as it makes the `ExceptionHandler` simpler (https://github.com/mrc-ide/hint/pull/215/files#diff-2a06c226eeb55efc9a48ced0a230f53bL41) and also removes that `postResetPassword.password` prefix on the error message https://github.com/mrc-ide/hint/pull/215/files#diff-fdb106bec0ed7f070cacea437a573925L108